src/screens/CompanyScreen.js

```diff
@@ -28,19 +28,21 @@ import {
 } from '../components';
 import { getQuery, QUERY_TYPES } from '../queries';
 import { graphqlFetchPolicy, refreshTimeFor } from '../helpers';
+import { useMatomoTrackScreenView } from '../hooks';
 
 export const CompanyScreen = ({ navigation }) => {
   const [refreshTime, setRefreshTime] = useState();
   const { isConnected, isMainserverUp } = useContext(NetworkContext);
   const { orientation, dimensions } = useContext(OrientationContext);
   const globalSettings = useContext(GlobalSettingsContext);
   const [refreshing, setRefreshing] = useState(false);
+  const { MATOMO_TRACKING, REFRESH_INTERVALS } = consts;
 
   useEffect(() => {
     const getRefreshTime = async () => {
```

Comment:
here could be a `useCallback` as well

Reply:
thanks, needs to be refactored with #121